pkg/remoteconfig/templates/avalanche-aliases.tmpl

```diff
@@ -0,0 +1,3 @@
+{
+ "{{.BlockChainID}}": ["{{join .Aliases "\", \""}}"]
```

Comment:
sorry, we did not understood here. what I wanted to suggest is this: on wiz for example, you can deploy
more than one blockchain to a given cluster. first time you use wiz, it creates the deploy and deploys
a blockchain. second time you use wiz, with the same cluster as arg, it deploys a SECOND blockchain
on the cluster.
in the previous review, I was asking the implementation of alias to support aliasing the first deployed blockchain,
also the second deployed blockchain. the implementation seems to just provide alias for one blockchain

Reply:
it will work this way as this file is not directly created but merged with previos version on the remote host

Reply:
this is supported as we merge new alias data with previous alias file if it's found on the remote.
this approach will be also useful for sdk